addons/l10n_co_pos_details/models/pos_config.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    l10n_co_pos_serial_number = fields.Char(string="POS Serial Number")
+    country_code = fields.Char(related='company_id.country_id.code')
```

Comment:
@jco-odoo it looks good to me, but aren't you using the country_id of the chart template in other localisation (instead of the one of the company? even if it is probably always the same
"company_id.chart_template_id.country_id.code" ?

Reply:
@pimodoo From v15, we have company_id.account_fiscal_country_id (or something) which gets the country copied from the chart of accounts.